input/docs/tutorials/using-cake-pre-release.md

```diff
@@ -0,0 +1,29 @@
+Order: 35
+---
+
+Cake uses [MyGet](https://www.myget.org/) as a NuGet feed for testing and pre-release builds. With these pre-release builds the next version of Cake can be accessed and utilized for getting the latest features or testing addins or build scripts to know if the next release will be safe when you need to upgrade. 
+
+# Using the pre-release version of Cake
+
+These instructions you are using the nuget CLI as done in the [default bootstrapper](https://github.com/cake-build/resources/blob/develop/build.ps1). We will start by editing the following line in the bootstrapper. This will add the MyGet source.
```

Comment:
@phillipsj think there is something missing here....  Should it be `These instructions assume you are using...`

Reply:
I have fixed the typo and rebased on develop.